Cargo.toml

```diff
@@ -179,39 +179,43 @@ codegen-units = 1
 rpath = true
 
 [patch.crates-io]
-# Fork and rename to use "OG" dalek-cryptography.
-bulletproofs-og = { git = "https://github.com/mobilecoinfoundation/bulletproofs.git", rev = "675330c754f28876dbf94fc303fe73666cf8f8f4" }
+# Fork to use latest dependencies.
+bulletproofs = { git = "https://github.com/remoun/bulletproofs.git", branch = "mobilecoin" }
```

Comment:
This should be done in the mobilecoin fork not your personal fork

Reply:
Fair enough. I will update this PR before merging it, after those PRs are merged.